src/components/visualizations/chart/highcharts/comboConfiguration.ts

```diff
@@ -1,11 +1,32 @@
 // (C) 2007-2019 GoodData Corporation
+import get = require("lodash/get");
+import isEqual = require("lodash/isEqual");
 import { MAX_POINT_WIDTH } from "./commonConfiguration";
 import { LINE_WIDTH } from "./lineConfiguration";
+import { IChartConfig } from "../../../../interfaces/Config";
+import { VisualizationTypes } from "../../../../constants/visualizationTypes";
 
-export function getComboConfiguration() {
+const { COLUMN, LINE } = VisualizationTypes;
+
+function getDefaultChartType(config?: IChartConfig) {
```

Comment:
we should have some unit tests for this `getDefaultChartType`

Reply:
updated :)